client/dashboard/components/code-highlighter/style.scss

```diff
@@ -0,0 +1,17 @@
+.github-workflow-code-highlighter {
```

Comment:
If it's at component level, do we really want this `github-workflow-` prefix?

Reply:
You're right, this is not important in this case IMO, removed in d186ed544a9f68c8aee55b7e4fa74e835b27704d